bofire/strategies/doe/utils.py

```diff
@@ -488,3 +496,295 @@ def nchoosek_constraints_as_bounds(
     bounds = [(b[0], b[1]) for b in bounds]
 
     return bounds
+
+
+def discrete_to_relaxable_domain_mapper(domain: Domain) -> Domain:
```

Comment:
Do we really need new feature types, or can then strategy just interpret them in a different manner?